api/explorer/api/gremlin.py

```diff
@@ -108,17 +108,19 @@ def connect(self, url: str, headers: dict | None = None) -> None:
             headers (dict | None, optional): Extra HTTP headers to include in
                 requests. Defaults to None.
         """
-        if self._client is None:
-            self._client = client.connect(
-                uri=url,
-                additional_headers=headers,
-                open_timeout=self.open_timeout,
-                close_timeout=self.close_timeout,
-                ssl_context=self.ssl_context,
-                logger=self.connection_logger,
-            )
-            self._last_connect_url = url
-            self._last_connect_headers = headers
+        if self._client is not None:
+            self.close()
```

Comment:
don't we want to prevent a new connetion if the connection has not been closed before?

Reply:
FYI - the base class implementation is [here](https://github.com/apache/tinkerpop/blob/0075cee46e13ee188ed1a25bb1235a4df2db3a9a/gremlin-python/src/main/python/gremlin_python/driver/transport.py). Unfortunately there is not documentation.

Reply:
makes sense, could you add a comment with a reference to relevant information? (maybe even just this discussion)